### PR TITLE
Fix for parsing very long string which was causing obd data to not be shown

### PR DESCRIPTION
**Pull request by landoulsi**

### CODE REVIEW

File: library/src/main/java/com/fr3ts0n/ecu/prot/obd/ObdProt.java [hunk 1]
```diff
@@ -715,6 +715,17 @@ public synchronized int handleTelegram(char[] buffer)
         return (result);
     }
 
+    private String getPayLoadString(char[] buffer) {
```
Comment (landoulsi) [inline library/src/main/java/com/fr3ts0n/ecu/prot/obd/ObdProt.java]: This is the actual fix

**Comment by fr3ts0n** [comment]
I agree to the functional change in ObdProt.
Please could you remove the changes in gradle files, since they create additional dependencies which are not really needed.
These may violate the FDroid builds.
Thx

**Comment by fr3ts0n** [comment]
Unfortunately we could not get your PR #91 cleaned up.
Changes are now implemented from my side with commit 67e71c936673aede40b0213e3d0f3f9dcc1a34ac.

File: androbd/build.gradle [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 28
     buildToolsVersion '28.0.3'
     defaultConfig {
         applicationId 'com.fr3ts0n.ecu.gui.androbd'
         minSdkVersion 15
-        targetSdkVersion 25
+        targetSdkVersion 26
```
[the remark this answers is not in the document]
Comment (puneetkansal04) [reply androbd/build.gradle:11]: So if needs it to be update. How can we do this?

**Pull request closed without merging**